src/data/models/Happiness.ts

```diff
@@ -1,10 +1,11 @@
+import { User } from "./User";
+
 export interface Happiness {
   id: number;
-  user_id: number;
   value: number;
   comment: string;
   timestamp: string;
-  // author: User;
+  author: User;
```

Comment:
has this been merged into the backend?

Reply:
let's just pretend it is for now, Aaron should have a test backend we can use ready soon.